code/modules/vending/_vending.dm

```diff
@@ -861,6 +867,15 @@
 /atom/movable/proc/post_tilt()
 	return
 
+/atom/movable/proc/vendor_can_crush_item(atom/atom_target)
```

Comment:
This isn't a particularly good name since

A) it doesn't apply to just vendors
B) it doesn't apply to items, but machineries and structures. It's also made to work on the atom level

make it 'is_crushable_atom' or 'check_atom_crushability' or something instead

Also also, this doesn't need to live on atom/movable. Make it a global proc, so just /proc/global_proc with no prepended path